### PR TITLE
Improve Italian translation

### PR DESCRIPTION
**Pull request by iomega11**
After working on #433 I noticed some new features didn't have their Italian translation. Here they are !

### CODE REVIEW

File: Mifare Classic Tool/app/src/main/res/values-it/strings.xml [hunk 1]
```diff
@@ -586,5 +592,6 @@
     <string name="hint_int_423">es: 423</string>
     <string name="hint_custom_sector_count">es: 16 per 1K</string>
     <string name="hint_custom_retry_authentication_count">Numero di tentativi</string>
+    <string name="hint_key">Esadecimale, 6 byte per linea</string>
```
Comment (ErikPelli) [inline Mifare Classic Tool/app/src/main/res/values-it/strings.xml:595]:
```suggestion
    <string name="hint_key">Esadecimale, 6 byte per riga</string>
```

File: Mifare Classic Tool/app/src/main/res/values-it/strings.xml [hunk 2]
```diff
@@ -302,11 +304,13 @@
     <string name="action_generic_converter">Convertitore del tipo di dati</string>
     <string name="action_multi_purpose_converter">Convertitore multiuso</string>
     <string name="action_cyber_chef">Cyber Chef</string>
+    <string name="action_open_clone_uid_tool">Apri lo strumento per clonare l\'UID</string>
+    <string name="action_hide_identical_sectors">Hide identical sectors</string>
```
Comment (ErikPelli) [inline Mifare Classic Tool/app/src/main/res/values-it/strings.xml:308]:
```suggestion
    <string name="action_hide_identical_sectors">Nascondi i settori identici</string>
```

File: Mifare Classic Tool/app/src/main/res/values-it/strings.xml [hunk 3]
```diff
@@ -199,6 +199,8 @@
     <string name="text_missing_files_update">Hai perso i file dopo l\'aggiornamento alla versione 4.0.0 o superiore? Fai riferimento
         al punto \"1.2 Data Storage\" nella sezione \"Informazioni\".</string>
     <string name="text_more_converter">Più convertitori (online)</string>
+    <string name="text_enter_file_name">Inserisci un nome per il file</string>
+    <string name="text_difference_between_dumps">Differenza tra dumps</string>
```
Comment (ErikPelli) [inline Mifare Classic Tool/app/src/main/res/values-it/strings.xml:203]:
```suggestion
    <string name="text_difference_between_dumps">Differenza tra i dump</string>
```

**Comment by ikarus23** [comment]
Thanks!

**Comment by ikarus23** [comment]
The build fails.
```
/home/runner/work/MifareClassicTool/MifareClassicTool/Mifare Classic Tool/app/src/main/res/values-it/strings.xml:382:4: Failed to flatten XML for resource 'info_keys_loaded_please_wait' with error: Invalid unicode escape sequence in string
```
I guess the `un po' di` in `info_keys_loaded_please_wait` should be `un po\' di`

**Pull request merged**